lib/controller/page/type/page.summary/page.summary.controller.unit.spec.js

```diff
@@ -304,12 +327,30 @@ test.skip('TEAM PDF: it attaches PDF contents to email submission attachments',
   t.end()
 })
 
-test.skip('USER PDF: it attaches PDF contents to email submission attachments', async t => {
-  getUserDataPropertyStub.resetHistory()
-  getUserDataPropertyStub.resetBehavior()
+test('USER PDF: it attaches PDF contents to email submission attachments', async t => {
+  const pageSummaryController = proxyquire('./page.summary.controller', {
+    '../../../../presenter/pdf-payload': {pdfPayload: pdfPayloadStub},
+    '../../../../middleware/routes-output/submission-data-with-labels': {submissionDataWithLabels: submissionDataWithLabelsStub},
+    '../../../../page/check-submits/check-submits': {checkSubmits: checkSubmitsStub},
+    '../../../../constants/constants': {
+      SERVICE_OUTPUT_EMAIL: 'bob@gov.uk'
+    }
+  })
 
-  getInstancePropertyStub.withArgs('service', 'emailTemplateTeam').returns('user {firstname} submitted!')
-  getUserDataPropertyStub.withArgs('attachUserSubmission').returns(true)
+  const userdata = getUserDataMethods({})
+  await pageSummaryController.postValidation({}, userdata)
+
+  const submissions = submitterClientSpy.getCall(0).args[0].submissions
+  const emailEntries = submissions.filter(s => s.type === 'email')
+  const pdfData = emailEntries[0].attachments[0].pdf_data
+
+  t.deepEquals(pdfData, {some: 'PDF contents'})
+  submitterClientSpy.resetHistory()
+  t.end()
+})
+
+test('team email body can be overiden to inculed a rendered template', async t => {
```

Comment:
Some typos here but also not sure what it means to include a rendered template?

"team email body can be overridden" should be sufficient no?